frontend/connection/src/constants.ts

```diff
@@ -39,8 +39,13 @@ export const HOST_CONFIG_PATH = "_stcore/host-config"
 /**
  * Min and max wait time between pings in millis.
  */
-export const PING_MINIMUM_RETRY_PERIOD_MS = 500
-export const PING_MAXIMUM_RETRY_PERIOD_MS = 1000 * 60
+export const PING_MINIMUM_RETRY_PERIOD_MS = 100
+export const PING_MAXIMUM_RETRY_PERIOD_MS = 2000
+
+/**
+ * Max number of times we retry pinging the server before we show an error.
+ */
+export const MAX_RETRIES_BEFORE_CLIENT_ERROR = 6
```

Comment:
Moved this into this file to match the other connection-related constants.

Reply:
We should probably also use this constant instead in `ConnectionManager` here:
https://github.com/streamlit/streamlit/blob/ec9d236e6ccc865cd81b5dc39d4ab7b456235ce3/frontend/connection/src/ConnectionManager.ts#L27-L34